ui/raidboss/data/raidboss_manifest.txt

```diff
@@ -397,6 +397,7 @@
 06-ew/dungeon/the_aitiascope.txt
 06-ew/dungeon/the_dead_ends.ts
 06-ew/dungeon/the_dead_ends.txt
+06-ew/dungeon/the_excitatron_6000.ts
```

Comment:
If you're thinking of adding more of these, maybe we need a `maps` subdirectory for each expansion? What do you think?

(Still not sure what to do about carnivale, honestly, since there's only one of them.)

Reply:
I think there's at most 3 treasure dungeons per expansion (so far)?  I'll probably collect info for the new one that was just announced when it comes out, but I don't know if I'll go back and do the older ones.  It's not something you can just queue into at-will and the RNG of getting to the final floor makes it a bit of a pain to collect data on.

Reply:
My thought for Masked Carnivale was maybe re-parent it under the General triggers category.

Reply:
That makes sense to me!

Reply:
> I think there's at most 3 treasure dungeons per expansion (so far)? I'll probably collect info for the new one that was just announced when it comes out, but I don't know if I'll go back and do the older ones. It's not something you can just queue into at-will and the RNG of getting to the final floor makes it a bit of a pain to collect data on.

Also yeah, I wasn't expecting you to go back and do others, but if one exists then other people (or me) might imagine adding more, so it's good to have an idea of structure for where those might fit in.

Reply:
Moved carnivale in #5068

Reply:
I moved this to a `treasure_dungeon` sub-directory; should be unambiguous what goes there for future additions.

Reply:
Sorry to nitpick, but `treasure_dungeon` is just a really long name.  Is `map` too ambiguous for you?

Reply:
I suppose not, since zone is mostly how we refer to areas.